app/controllers/inventories_controller.rb

```diff
@@ -0,0 +1,45 @@
+# rubocop:disable Lint/RescueException
+
+class InventoriesController < ApplicationController
```

Comment:
- [ ] Change styling to this ubtton, users cant really see it
![Screenshot from 2022-04-30 17-21-13](https://user-images.githubusercontent.com/62705947/166111688-2b58c4de-6c59-4b24-9c8a-0d4d0c3de746.png)